recipe/build.sh

```diff
@@ -31,7 +31,7 @@ if [[ $mpi == "openmpi" ]]; then
   export OMPI_MCA_plm=isolated
   export OMPI_MCA_rmaps_base_oversubscribe=yes
   export OMPI_MCA_btl_vader_single_copy_mechanism=none
-  export WITH_SUPERLU_DIST=0
+  export WITH_SUPERLU_DIST=1
```

Comment:
Perhaps we should just get rid of this `WITH_SUPERLU_DIST` variable? After all, now we build with SuperLU_Dist for both MPICH and Open MPI. Could you please update the recipe accordingly?

Reply:
I removed the environment variable `WITH_SUPERLU_DIST`